### DISCUSSION TITLE
How to learn and contribute to this project?

### DISCUSSION THREAD

**Discussion by yaten2302** [lawrencehook/remove-youtube-suggestions#100] [Q&A] [answered]
Hi, I'm a beginner programmer. I know HTML, CSS, Tailwind CSS and currently learning JavaScript.

I want to contribute to this project, but, I am a beginner and I haven't made such advanced projects, can anyone please guide me that how and from where should I learn, so that I can contribute to this project🙏🙏

**Comment by lawrencehook** [comment, marked as answer]
Hello, thanks for making this conversation. Here's my attempt at a brief overview, if helpful. [This file](https://github.com/lawrencehook/remove-youtube-suggestions/blob/main/src/shared/main.js) stores all the settings. The content-script files run on YouTube pages, and edit the site according to the settings. The options/ files relate to the extension's settings page, with the sections and toggles for each setting.
